platform/android/export/export_plugin.cpp

```diff
@@ -62,158 +62,158 @@
-	"BIND_VPN_SERVICE",
-	"BIND_WALLPAPER",
-	"BLUETOOTH",
-	"BLUETOOTH_ADMIN",
-	"BLUETOOTH_PRIVILEGED",
-	"BRICK",
-	"BROADCAST_PACKAGE_REMOVED",
-	"BROADCAST_SMS",
-	"BROADCAST_STICKY",
-	"BROADCAST_WAP_PUSH",
-	"CALL_PHONE",
-	"CALL_PRIVILEGED",
-	"CAMERA",
-	"CAPTURE_AUDIO_OUTPUT",
-	"CAPTURE_SECURE_VIDEO_OUTPUT",
-	"CAPTURE_VIDEO_OUTPUT",
-	"CHANGE_COMPONENT_ENABLED_STATE",
-	"CHANGE_CONFIGURATION",
-	"CHANGE_NETWORK_STATE",
-	"CHANGE_WIFI_MULTICAST_STATE",
-	"CHANGE_WIFI_STATE",
-	"CLEAR_APP_CACHE",
-	"CLEAR_APP_USER_DATA",
-	"CONTROL_LOCATION_UPDATES",
-	"DELETE_CACHE_FILES",
-	"DELETE_PACKAGES",
-	"DEVICE_POWER",
-	"DIAGNOSTIC",
-	"DISABLE_KEYGUARD",
-	"DUMP",
-	"EXPAND_STATUS_BAR",
-	"FACTORY_TEST",
-	"FLASHLIGHT",
-	"FORCE_BACK",
-	"GET_ACCOUNTS",
-	"GET_PACKAGE_SIZE",
-	"GET_TASKS",
-	"GET_TOP_ACTIVITY_INFO",
-	"GLOBAL_SEARCH",
-	"HARDWARE_TEST",
-	"INJECT_EVENTS",
-	"INSTALL_LOCATION_PROVIDER",
-	"INSTALL_PACKAGES",
-	"INSTALL_SHORTCUT",
-	"INTERNAL_SYSTEM_WINDOW",
-	"INTERNET",
-	"KILL_BACKGROUND_PROCESSES",
-	"LOCATION_HARDWARE",
-	"MANAGE_ACCOUNTS",
-	"MANAGE_APP_TOKENS",
-	"MANAGE_DOCUMENTS",
-	"MANAGE_EXTERNAL_STORAGE",
-	"MASTER_CLEAR",
-	"MEDIA_CONTENT_CONTROL",
-	"MODIFY_AUDIO_SETTINGS",
-	"MODIFY_PHONE_STATE",
-	"MOUNT_FORMAT_FILESYSTEMS",
-	"MOUNT_UNMOUNT_FILESYSTEMS",
-	"NFC",
-	"PERSISTENT_ACTIVITY",
-	"POST_NOTIFICATIONS",
-	"PROCESS_OUTGOING_CALLS",
-	"READ_CALENDAR",
-	"READ_CALL_LOG",
-	"READ_CONTACTS",
-	"READ_EXTERNAL_STORAGE",
-	"READ_FRAME_BUFFER",
-	"READ_HISTORY_BOOKMARKS",
-	"READ_INPUT_STATE",
-	"READ_LOGS",
-	"READ_MEDIA_AUDIO",
-	"READ_MEDIA_IMAGES",
-	"READ_MEDIA_VIDEO",
-	"READ_MEDIA_VISUAL_USER_SELECTED",
-	"READ_PHONE_STATE",
-	"READ_PROFILE",
-	"READ_SMS",
-	"READ_SOCIAL_STREAM",
-	"READ_SYNC_SETTINGS",
-	"READ_SYNC_STATS",
-	"READ_USER_DICTIONARY",
-	"REBOOT",
-	"RECEIVE_BOOT_COMPLETED",
-	"RECEIVE_MMS",
-	"RECEIVE_SMS",
-	"RECEIVE_WAP_PUSH",
-	"RECORD_AUDIO",
-	"REORDER_TASKS",
-	"RESTART_PACKAGES",
-	"SEND_RESPOND_VIA_MESSAGE",
-	"SEND_SMS",
-	"SET_ACTIVITY_WATCHER",
-	"SET_ALARM",
-	"SET_ALWAYS_FINISH",
-	"SET_ANIMATION_SCALE",
-	"SET_DEBUG_APP",
-	"SET_ORIENTATION",
-	"SET_POINTER_SPEED",
-	"SET_PREFERRED_APPLICATIONS",
-	"SET_PROCESS_LIMIT",
-	"SET_TIME",
-	"SET_TIME_ZONE",
-	"SET_WALLPAPER",
-	"SET_WALLPAPER_HINTS",
-	"SIGNAL_PERSISTENT_PROCESSES",
-	"STATUS_BAR",
-	"SUBSCRIBED_FEEDS_READ",
-	"SUBSCRIBED_FEEDS_WRITE",
-	"SYSTEM_ALERT_WINDOW",
-	"TRANSMIT_IR",
-	"UNINSTALL_SHORTCUT",
-	"UPDATE_DEVICE_STATS",
-	"USE_CREDENTIALS",
-	"USE_SIP",
-	"VIBRATE",
-	"WAKE_LOCK",
-	"WRITE_APN_SETTINGS",
-	"WRITE_CALENDAR",
-	"WRITE_CALL_LOG",
-	"WRITE_CONTACTS",
-	"WRITE_EXTERNAL_STORAGE",
-	"WRITE_GSERVICES",
-	"WRITE_HISTORY_BOOKMARKS",
-	"WRITE_PROFILE",
-	"WRITE_SECURE_SETTINGS",
-	"WRITE_SETTINGS",
-	"WRITE_SMS",
-	"WRITE_SOCIAL_STREAM",
-	"WRITE_SYNC_SETTINGS",
-	"WRITE_USER_DICTIONARY",
+	PNAME("ACCESS_CHECKIN_PROPERTIES"),
+	PNAME("ACCESS_COARSE_LOCATION"),
+	PNAME("ACCESS_FINE_LOCATION"),
```

Comment:
Does this mean that users will be translating `ACCESS_CHECKIN_PROPERTIES` or the humanized `Access Checkin Properties`? The former is actual code and should likely be kept as is but the user facing Title Cased string would be nice to localize.

Reply:
The translation applies to humanized version.